### PR TITLE
fixing bug city, village, district not appear on detail applicant invitation

### PR DESCRIPTION
**Pull request by rachadiannovansyah**
Refer to task -> https://trello.com/c/0yt57XiJ/201-tidak-ada-kota-kecamatan-dan-kelurahan-di-detail-peserta-di-dalam-kegiatan
Solved by reading this -> https://laravel.com/docs/8.x/eloquent-relationships#lazy-eager-loading

Overview:
- Fixing bug city, district, village not appear on detail applicant invitation

Note:
- Tested

cc: mas @yohang88

### CODE REVIEW

File: app/Http/Controllers/Rdt/RdtEventParticipantDetailController.php [hunk 1]
```diff
@@ -2,16 +2,19 @@
...
 
-use App\Http\Controllers\Controller;
-use Illuminate\Http\Request;
 use App\Entities\RdtInvitation;
+use App\Http\Controllers\Controller;
 use App\Http\Resources\RdtInvitationResource;
 
 class RdtEventParticipantDetailController extends Controller
 {
     public function __invoke($id)
```
Comment (ayocodingit) [inline app/Http/Controllers/Rdt/RdtEventParticipantDetailController.php:11]: ini gak coba menggunakan route binding kang @rachadiannovansyah

Comment (rachadiannovansyah) [reply app/Http/Controllers/Rdt/RdtEventParticipantDetailController.php]: waiyaa kode warisan :grin: 
done kang :+1:

**Comment by rachadiannovansyah** [comment]
done cek cekkk :grin: cc : mas @yohang88 @ayocodingit

**Review by ayocodingit — APPROVED** [review]
nice kang @rachadiannovansyah

File: app/Http/Controllers/Rdt/RdtInvitationLabResultUpdateController.php [hunk 2]
```diff
@@ -8,9 +8,9 @@
 
 class RdtInvitationLabResultUpdateController extends Controller
 {
-    public function __invoke($id, Request $request)
+    public function __invoke(RdtInvitation $rdtInvitation, Request $request)
     {
-        $rdtInvitation  = RdtInvitation::findOrFail($id);
+        // implement model route binding
```
Comment (ayocodingit) [inline app/Http/Controllers/Rdt/RdtInvitationLabResultUpdateController.php:13]: comment nya boleh di hapus kang @rachadiannovansyah

File: app/Http/Controllers/Rdt/RdtInvitationResetController.php [hunk 3]
```diff
@@ -7,10 +7,10 @@
 
 class RdtInvitationResetController extends Controller
 {
-    public function __invoke($id)
+    public function __invoke(RdtInvitation $rdtInvitation)
     {
-        $rdtInvitation  = RdtInvitation::findOrFail($id);
-        $rdtInvitation->update(['lab_code_sample' => null ,'attended_at' => null,'attend_location' => null]);
+        // implement model route binding
```
Comment (ayocodingit) [inline app/Http/Controllers/Rdt/RdtInvitationResetController.php:12]: comment nya boleh di hapus kang @rachadiannovansyah

File: app/Http/Controllers/Rdt/RdtInvitationResetController.php [hunk 4]
```diff
@@ -7,10 +7,10 @@
...
+        $rdtInvitation->update(['lab_code_sample' => 'null' ,'attended_at' => null,'attend_location' => null]);
```
Comment (ayocodingit) [inline app/Http/Controllers/Rdt/RdtInvitationResetController.php:13]: kenapa null nya string yah

Comment (ayocodingit) [reply app/Http/Controllers/Rdt/RdtInvitationResetController.php:13]: sama spasi nya minor PSR 12 yah

Comment (rachadiannovansyah) [reply app/Http/Controllers/Rdt/RdtInvitationResetController.php]: nggak ko kang udh aman, ini setiap mau di push di run dulu `PHP Sniffernya`

Comment (ayocodingit) [reply app/Http/Controllers/Rdt/RdtInvitationResetController.php]: setelah comma spasi kang @rachadiannovansyah

Comment (ayocodingit) [reply app/Http/Controllers/Rdt/RdtInvitationResetController.php]: sipp mantap kang @rachadiannovansyah

Comment (rachadiannovansyah) [reply app/Http/Controllers/Rdt/RdtInvitationResetController.php]: seppp kang, btw `phpcbf` ga aware sama comma ya *cmiiw
itu di fix pake `phpfmt` extension vscode :grin:

Comment (ayocodingit) [reply app/Http/Controllers/Rdt/RdtInvitationResetController.php]: kombinasikan saja kang @rachadiannovansyah :+1:

**Comment by rachadiannovansyah** [comment]
up up mas @yohang88 :grin:

**Review by yohang88 — APPROVED** [review]

**Pull request merged**